Raven.Database/Impl/BackgroundTaskExecuter/RavenThreadPool.cs

```diff
@@ -104,7 +105,17 @@ private void ExecuteLongRunningTask(ThreadData threadData, Action longRunningAct
             }
             catch (Exception e)
             {
-                logger.FatalException("Error from long running task in thread pool, task terminated, this is VERY bad", e);
+                const string error = "Error from long running task in thread pool, task terminated, this is very bad";
+                logger.FatalException(error, e);
+                database.AddAlert(new Alert
+                {
+                    AlertLevel = AlertLevel.Error,
+                    CreatedAt = DateTime.UtcNow,
+                    Title = error,
+                    UniqueKey = error,
+                    Message = $"StackTrace: {e.StackTrace}",
+                    Exception = e.Message
```

Comment:
Why are you using the message here? And not e.ToString()?
You are losing all the inner exceptions1